install-worker.sh

```diff
@@ -64,6 +64,8 @@ sudo yum install -y \
     unzip \
     wget
 
+if yum list installed | grep ec2-net-utils; then sudo yum remove ec2-net-utils -y -q; fi
+
```

Comment:
Thanks, it is true that this package causes problems with the AWS CNI. Could you please add a comment before this line to explain why we remove it.